### ISSUE TITLE
I have the problem of overfitting after I trained this model 

### ISSUE THREAD

**Issue by Hathaways** [rishabh30/Facial-Expression-Recognition#3]
![2018-05-20 15-23-56](https://user-images.githubusercontent.com/22563647/40276793-a2330846-5c45-11e8-99ea-187010a01a09.png)

**Comment by rishabhjainps** [comment]
Yes there is problem of over-fitting in the model if trained for whole 125 epochs.

- Dropout is used to solve over-fitting but here it is not solving the problem fully .
- Also tried using regularization of weights to solve it but still got the same problem.

At last I used early stopping to prevent the model to over-fit the training set and still got around same validation and testing accuracy.

Open for other suggestions.
